pkg/ocm/ocm_config.go

```diff
@@ -9,6 +9,10 @@ import (
 	sdk "github.com/openshift-online/ocm-sdk-go"
 )
 
+// ocmConfigFile is the path refers to existing ocm config file
+// set by the ocm command by calling ocm login
+var ocmConfigFile string
```

Comment:
Based on the deleted comment that this should only be used locally this is probably ok.

However if this ever gets used in production with a long-running CAD process this could potentially lead to weird race-conditions if multiple investigations try to set this to different values and might need to be wrapped in a `sync.Mutex` to change.

(Or even better, not be a global variable but be set per-investigation)

Reply:
The config should be used only locally not in prod. So this is not an issue.